spec/dummy/config/environments/development.rb

```diff
@@ -27,9 +27,6 @@
     config.cache_store = :null_store
   end
 
-  # Store uploaded files on the local file system (see config/storage.yml for options)
-  config.active_storage.service = :local
```

Comment:
why was this?

Reply:
we don't need active storage in the dummy app, and it was making the dummy app not be able to run